lib/bot.js

```diff
@@ -37,6 +41,14 @@ class Bot {
     this.ircStatusNotices = options.ircStatusNotices;
     this.announceSelfJoin = options.announceSelfJoin;
 
+    // IRC nicks to ignore (case insensitive)
+    this.ircIgnoreUsers = options.ircIgnoreUsers || [];
```

Comment:
could we change the config to be an object?

```json
"igoreUsers": {
  "irc":" [],
  "discord": [],
}
```

Reply:
Good idea, fixed.